packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -470,17 +470,22 @@ module.exports.subscriptionsStripeSubscriptionItemValidator = isA
 module.exports.subscriptionsStripeSubscriptionValidator = isA
   .object({
     id: isA.string().required(),
-    cancel_at: isA.number().optional(),
-    canceled_at: isA.number().optional(),
+    cancel_at: isA.alternatives(isA.number(), isA.any().allow(null)),
+    canceled_at: isA.alternatives(isA.number(), isA.any().allow(null)),
```

Comment:
Isn't `isA.number()` a subset of `isA.any().allow(null)`?  I'm trying to understand why we'd need both.

Reply:
I don't think so - I think it's either a number or a null. The `any()` validator starts generic, but `allow()` restricts it to just null. We use this construction elsewhere in validators here